README.md

```diff
@@ -17,6 +17,26 @@
 composer require smartbooster/authentication-bundle
 ```
 
+### Configure block
+
+``` yaml
+    # sonata.yml
+    sonata_admin:
+        templates:
+            user_block: "@SmartAuthentication/block/user.html.twig"
+```
+
+### Configure admin extension
+
+``` yaml
+    # sonata.yml
```

Comment:
So you can load this directly like admin_extension.xml and the same for previous block by the way

https://github.com/prestaconcept/PrestaCMSCoreBundle/blob/master/DependencyInjection/PrestaCMSCoreExtension.php

Reply:
i fix the configuration loading using this https://symfony.com/doc/3.4/bundles/prepend_extension.html

now there is no need to add it on the main config.yml